internal/services/recoveryservices/site_recovery_replicated_vm_resource_test.go

```diff
@@ -1794,8 +1794,8 @@ resource "azurerm_orchestrated_virtual_machine_scale_set" "test" {
 
   source_image_reference {
     publisher = "Canonical"
-    offer     = "UbuntuServer"
-    sku       = "16.04-LTS"
+    offer     = "0001-com-ubuntu-server-focal"
+    sku       = "20_04-lts"
```

Comment:
this is using `20` instead of `22`, because the kernel version 6.x in 22 image is not supported for now.

Reply:
the error is like
```
        Error: creating replicated vm repl-231009085411405139 (vault acctest-vault-231009085411405139): polling after Create: polling failed: the Azure API returned the following error:
        Status: "Unknown"
        Code: "151113"
        Message: "This version of mobility service doesn't support the operating system kernel version (6.2.0-1014-azure) running on the source machine. Please refer the list of operating systems supported by Azure Site Recovery : https://aka.ms/a2a_supported_linux_os_versions
        Activity Id: ""
```